user/unified-platform.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Travis CI Single Platform"
+---
+
+On May 2nd, 2018 Travis CI integration open source and private projects onto a unified platform on **travis-ci.com**!
+
+This means you can manage public and private repositories from one domain, allow open source projects the ability to add extra concurrency, and allow our team to maintain and improve a single CI platform.
+
+We are implementing these changes with GitHub Apps, which not only provides a richer integration with GitHub, but also allows for many amazing features we plan to roll out in the next few months.
```

Comment:
I'd remove "but also allows for many amazing features we plan to roll out in the next few months" in the docs here. I'm not sure it brings much value